packages/@react-stately/tree/src/useTreeState.ts

```diff
@@ -31,6 +31,9 @@ export interface TreeState<T> {
   /** Toggles the expanded state for an item by its key. */
   toggleKey(key: Key): void,
 
+  /** Sets the expanded state for a set of keys. */
```

Comment:
This makes it sound like you're setting the expanded state for the keys you pass in without affecting the ones that are already expanded.

```suggestion
  /** Replaces the set of expanded keys. */
```